tests/ui/append_instead_of_extend.rs

```diff
@@ -0,0 +1,32 @@
+#![warn(clippy::append_instead_of_extend)]
```

Comment:
You can add the following as the first line in the file to also test the automated application of the suggestion:
```rs
// run-rustfix
```

You only need to run the test afterwards once and bless the results. The rest should be taken care of for you :)